src/sbt-test/skip-windows/caching/test

```diff
@@ -302,7 +302,7 @@ $ copy-file files/Valid.scala src/main/scala/Valid.scala
 > set scalafixDependencies in ThisBuild := Nil
 > scalafix --check ProcedureSyntax
 > reload plugins
-> 'set dependencyOverrides += "ch.epfl.scala" % "scalafix-interfaces" % "0.9.13"' // anything else than current
+> 'set dependencyOverrides += "ch.epfl.scala" % "scalafix-interfaces" % "0.9.17+7-a1b6d972-SNAPSHOT"' // anything else than current
```

Comment:
prevents `java.lang.NoSuchMethodError: scalafix.interfaces.Scalafix.fetchAndClassloadInstance(Ljava/lang/String;Ljava/util/List;)Lscalafix/interfaces/Scalafix;` - it's expected that the user can't downgrade the scalafix version